spyder/utils/syntaxhighlighters.py

```diff
@@ -422,6 +424,12 @@ def __init__(self, parent, font=None, color_scheme='Spyder'):
         self.import_statements = {}
         self.found_cell_separators = False
         self.cell_separators = CELL_LANGUAGES['Python']
+        self.oetimer = QTimer()
+        self.oetimer.timeout.connect(self.emit_oe_changed)
+
+    @Slot()
+    def emit_oe_changed(self):
+        self.sig_outline_explorer_changed.emit()

```

Comment:
I would really prefer if we used longer descriptive names.

`emit_outline_changed` or even `emit_outline_explorer_changed` 

Same goes for the `self.oetimer` variable